modules/datagrid/launch/added/launch/prometheus.conf

```diff
@@ -19,6 +19,9 @@
 # Prometheus agent JAR path in our image
 PROMETHEUS_JAR=$(getfiles io/prometheus/jmx/jdg-7.3/)
 
+# Create a symlink to PROMETHEUS_JAR so it could be referenced easily in case JAVA_OPTS_APPEND env is used to turn on Prometheus
+ln -s ${PROMETHEUS_JAR} ${JBOSS_HOME}/jmx_prometheus_javaagent.jar
```

Comment:
Nitpick, but if this is for convenience why not just call it `prometheus.jar`

Reply:
`jmx_exporter.jar` would be most accurate.

Reply:
`jmx_prometheus_javaagent-0.3.1.redhat-00001.jar` would be the most accurate :stuck_out_tongue_winking_eye:

I'm happy with anything that abstracts away the version number.